docs/pages/develop/development-builds/development-workflows.mdx

```diff
@@ -79,9 +79,9 @@ These are a few examples of workflows to help your team get the most out of your
 
 ### PR previews
 
-You can set up your CI process to publish your project whenever a pull request is merged or updated and add a QR code that is used to view the change in a compatible development build.
+You can set up your CI process to publish an EAS Update whenever a pull request is updated and add a QR code that is used to view the change in a compatible development build.
 
-[expo-preview-action](https://github.com/expo/expo-preview-action) can be used to implement this workflow in your project using GitHub Actions or serve as a template in your CI of choice.
+Check out our [instructions for publishing app previews on pull requests](https://docs.expo.dev/eas-update/github-actions/#publish-previews-on-pull-requests) to implement this workflow in your project using GitHub Actions or serve as a template in your CI of choice.
```

Comment:
```suggestion
Check out our [instructions for publishing app previews on pull requests](/eas-update/github-actions/#publish-previews-on-pull-requests) to implement this workflow in your project using GitHub Actions or serve as a template in your CI of choice.
```

shouldn't use full url for docs otherwise it won't link to the correct page when in dev / staging environments

Reply:
This also forces the full re-render on navigation.

Reply:
thank you! don't know what I was thinking there 😅